pkgs/development/misc/gokrazy/default.nix

```
@@ -1,19 +1,27 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
 
 buildGoModule rec {
   pname = "gokrazy";
-  version = "unstable-2023-08-12";
+  version = "unstable-2024-09-27";
```

```suggestion
  version = "0-unstable-2024-09-27";
```
This is the new convention for unstable versions.